apps/i18n/common/en_us.json

```diff
@@ -1156,6 +1156,7 @@
   "enterYourAnswerHere": "Enter your answer here",
   "enterYourName": "Enter Your Name",
   "enterYourSchoolZip": "Enter your school's zip code:",
+  "enterYourSchoolZipPlaceholder": "30305",
```

Comment:
I don't think this needs to be translated. Though I'm curious where this zip came from and what best practices are here 🤔

Reply:
Oh good point! I just copied the zip from the Figma, but I can make it something else too if we want something more generic like "00000"?

Reply:
Let's do that and remove it from this file! It can just be hardcoded

Reply:
LOL that was my old zip, we should def change it to just 00000 😭